Sources/WalletSdk/MDoc.swift

```diff
@@ -84,17 +84,17 @@ public class BLESessionManager {
                 secKey = item as! SecKey
                 // swiftlint:enable force_cast
             case errSecItemNotFound:
-                self.callback.update(state: .error("Key not found"))
+                self.callback.update(state: .error(.generic("Key not found")))
```

Comment:
I think these are fine to keep as plain english, but we'll want to have more formal validation when constructing the mdoc, or key.